src/submission/validation-clinical/validation.ts

```diff
@@ -174,7 +181,10 @@ export const checkUniqueRecords = (
         { ...record, index: recordIndex },
         DataValidationErrors.FOUND_IDENTICAL_IDS,
         uniqueIdName,
-        { conflictingRows: sameIdentifiedRecordIndecies.filter(i => i !== recordIndex) },
+        {
+          conflictingRows: sameIdentifiedRecordIndecies.filter(i => i !== recordIndex),
+          useAllRecordValues,
```

Comment:
for my info what was the reason to add this flag to the info?

Reply:
its what I use to determine if the message says says `This row is identical to another row` or a specific message about the `ClinicalUniqueIdentifier`. There is no actual use case for this though